Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -5,56 +5,87 @@ import UIKit
 public extension CustomerIOImplementation {
     func setupAutoScreenviewTracking() {
         let selector1 = #selector(UIViewController.viewDidAppear(_:))
-        let selector2 = #selector(CustomerIOImplementation._swizzled_UIKit_viewDidAppear(_:))
+        let selector2 = #selector(UIViewController._swizzled_UIKit_viewDidAppear(_:))
         guard let originalMethod = class_getInstanceMethod(UIViewController.self, selector1) else {
             return
         }
-        guard let swizzleMethod = class_getInstanceMethod(CustomerIOImplementation.self, selector2) else {
+        guard let swizzleMethod = class_getInstanceMethod(UIViewController.self, selector2) else {
             return
         }
         method_exchangeImplementations(originalMethod, swizzleMethod)
     }
-
+}
+extension UIViewController {
+    var defaultScreenViewBody: ScreenViewData {
```

Comment:
Can you elaborate more, Stephen ? Do you mean that it should be `cio_UIViewController` ?

Reply:
I think Stephen is saying that any function or property that we add to `UIViewController` through extensions that the SDK uses internally, we should prefix the extension function name with `cio`. Therefore, preventing conflicts where our customers might have a function name that has the same name. 

Example: 

```swift
// before:
_swizzled_UIKit_viewDidAppear()
// after
cio_swizzled_UIKit_viewDidAppear()
```

We might also be able to solve the problem by making the extension functions/properties `internal` scope to avoid making them `public`.

Reply:
Ah ! Gotcha. Make sense ! Made the required changes.